README.md

```diff
@@ -1,99 +1,67 @@
-
-Clone the repository and follow these steps:
-
-### Pseudo-distributed deployment
-We provide docker-compose templates for a minimal standalone, pseudo-distributed Docker deployment, useful for local testing and development on a single host machine. 
-
-1. Create a default docker network  
-
-We need to make sure that all services deployed on our single host can communicate on the same docker network. Therefore, our provided docker-compose templates use a default external network 'fedn_default'. First, create this network: 
-
-````bash 
-docker network create fedn_default
-````
-
-2. Deploy the base services (Minio and MongoDB)  
-
-````bash 
-docker-compose -f config/base-services.yaml -f config/private-network.yaml up 
-````
-
-Make sure you can access the following services before proceeding to the next steps: 
- - Minio: http://localhost:9000
- - Mongo Express: http://localhost:8081
+## Getting started
 
-(You can update username and password for the services in config/base-services.yaml, default values: user=fedn_admin, password=password)
- 
-3. Start the Reducer  
+### Prerequisites
+- [Docker](https://docs.docker.com/get-docker)
+- [Docker Compose](https://docs.docker.com/compose/install)
+- [Python 3.8](https://www.python.org/downloads)
 
-Copy the settings config file for the reducer, 'config/settings-reducer.yaml.template' to 'config/settings-reducer.yaml'. You do not need to make any changes to this file to run the sandbox. To start the reducer service:
+### Quick start
+The quickest way to get started with FEDn is by trying out the [MNIST Keras example](examples/mnist-keras). If you just want to start FEDn and get a feeling of the UI and the architecture you can just start it in pseudo-distributed mode with Docker compose.
 
-````bash 
-docker-compose -f config/reducer.yaml -f config/private-network.yaml up 
-````
-
-Make sure that you can access the Reducer UI at https://localhost:8090. 
-
-4. Start a combiner  
-
-Copy the settings config file for the combiner, 'config/settings-combiner.yaml.template' to 'config/settings-combiner.yaml'. You do not need to make any changes to this file to run the sandbox. To start the combiner service and attach it to the reducer:
-
-````bash 
-docker-compose -f config/combiner.yaml -f config/private-network.yaml up 
-````
-
-You should now see the combiner listed in the 'Network' view in the UI.  
-
-### Train a federated model
-
-Training a federated model on the FEDn network involves: 
-
- 1. Configuring the network with a compute package (containing the code that will be distributed to clients)
- 2. Setting an initial model (untrained or pre-trained)
- 3. Attaching clients to the network 
-
-Follow the instruction here to set up your deployed network to train a model for digits classification using the MNIST dataset: 
-https://github.com/scaleoutsystems/examples/tree/main/mnist-keras
+```sh
+git clone https://github.com/scaleoutsystems/fedn.git
+cd fedn
+docker-compose up -d 
```

Comment:
I think some instructions for how to set the package and initial model is needed?

Reply:
Right! I removed it and just directed the user to the examples.